.github/ISSUE_TEMPLATE/3_Support_question.md

```diff
@@ -4,8 +4,12 @@ about: See https://astrodiscuss.usgs.gov for questions about using ISIS3 and oth
 
 ---
 
-We use GitHub issues only to discuss about ISIS3 bugs and new features. For
+We use GitHub issues only to discuss about ISIS3 bugs and new features. 
```

Comment:
This is really clunky wording. How about "We use Github issues solely for discussing bugs and new features"